joint_trajectory_controller/include/joint_trajectory_controller/joint_trajectory_controller.hpp

```diff
@@ -265,6 +265,12 @@ class JointTrajectoryController : public controller_interface::ControllerInterfa
 
   void read_state_from_hardware(JointTrajectoryPoint & state);
```

Comment:
When we are already fixing this. I would propose to rename the method to read_state_from_state_interfaces – it is a longer but clearer name. Read state from hardware is a bit unclear. Especially now with having chaining of state interfaces, this might be confusing.

```suggestion
  void read_state_from_state_interfaces(JointTrajectoryPoint & state);
```

Reply:
done